### ISSUE TITLE
Installation error on Ubuntu

### ISSUE THREAD

**Issue by jvhm** [stan-dev/shinystan#139]
Hi,

I'm trying to install shinystan on Ubuntu and I'm getting the following error:

> * installing *source* package ‘shinystan’ ...
>** package ‘shinystan’ successfully unpacked and MD5 sums checked
>** R
>** data
>*** moving datasets to lazyload DB
>** inst
>** preparing package for lazy loading
**>Error in dyn.load(file, DLLpath = DLLpath, ...) : 
>  ICU init failed: U_FILE_ACCESS_ERROR**
>ERROR: lazy loading failed for package ‘shinystan’
>* removing ‘/home/jvhm/R/x86_64-pc-linux-gnu-library/3.3/shinystan’
>
>The downloaded source packages are in
>	‘/tmp/RtmpfjMsIG/downloaded_packages’
>Warning message:
>In install.packages("shinystan", type = "source", repos = "http://cran.rstudio.com") :
>  installation of package ‘shinystan’ had non-zero exit status

I'm running R as root. Here's my system information:

> Kernel version: 4.4.0-38-generic
> Ubuntu version: 16.04.1 LTS
> R version: 3.3.1

I need this package in order to install "rstanarm", but I can't proceed because of this error.
Do you have any ideas about this issue?

Thanks in advance!

**Comment by jgabry** [comment]
Hmm, does this also happen if you install from the develop branch on github? You can try that like this:

```r
library("devtools")
install_github("stan-dev/shinystan", ref = "develop")
```

**Comment by jvhm** [comment]
Hi @jgabry!

Yes, it does happen even with the develop branch. But I also noticed it happened with another package I had to install (to be more specific, the "broom" package). So, after googling it a little bit more, I found out I had to install/update the "stringi" package (I don't know exactly why). After doing that, I didn't have this error anymore!

According to the "stringi" [docs](https://cran.r-project.org/web/packages/stringi/index.html):

> Owing to the use of the ICU library, the package provides R users with platform-independent functions known to Java, Perl, Python, PHP, and Ruby programmers

So, maybe it explains the error related to the ICU library...

Thanks!
